### PR TITLE
RB 250 C

### PR DESCRIPTION
**Pull request by arteck**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -1490,6 +1490,13 @@ const devices = [
         description: 'B22 Bulb RGBW',
         extend: generic.light_onoff_brightness_colortemp_colorxy,
     },
+    {
+        zigbeeModel: ['RB 250 C'],
+        model: 'RB 250 C',
+        vendor: 'Innr',
+        description: 'E27 Bulb RGBW',
```
Comment (Koenkk) [inline devices.js]: Isn't this a E14 bulb? (https://www.coolblue.nl/product/831417/innr-colour-e14-rb-250-c.html)

Comment (arteck) [reply devices.js]: oh sry yes.... its e14

Comment (arteck) [reply devices.js]: corr plz merge

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**